src/test/java/spoon/test/architecture/SpoonArchitectureEnforcerTest.java

```diff
@@ -328,6 +328,7 @@ public void testSpecPackage() throws Exception {
 		officialPackages.add("spoon.reflect.visitor");
 		officialPackages.add("spoon.reflect");
 		officialPackages.add("spoon.support.comparator");
+		officialPackages.add("spoon.support.comparator.cu");
```

Comment:
seems overkill to add a new package for those two small classes? spoon.support.comparator would be enough?